tools/toc.sh

```diff
@@ -9,7 +9,7 @@
 let ERROR=0
 
 # Find all markdown files
-for f in $(find . | grep .md); do
+for f in $(find doc/ boards/ kernel/ arch/ capsules/ chips/ userland/ -name "*.md"); do
```

Comment:
Yeah depending on git works until it doesn't. And it's never fun when it doesn't.